website/src/content/api-reference/commercetools-frontend-application-shell.mdx

```diff
@@ -73,6 +73,30 @@ export default EntryPoint;
 | `render`                         | `func`             |            (*)           | -       | The render function is called when the `<ApplicationShell>` is ready to render the actual application. This is the case when the required data (user, project) has been fetched and the application context has been initialized. It's **recommended to use the `children`** prop to benefit from a simpler setup.                                                                                                                                                                                                                                  |
 | `apolloClient`         | `ApolloClient`           |                       | -       | An optional instance of [ApolloClient](https://www.apollographql.com/docs/react/) to be used instead of the default one. This is usually the case when you need to configure the Apollo cache. See [`createApolloClient`](#createapolloclient).                                                                                                                                                                                                                                        |
 
+
+## ApplicationPageTitle
```

Comment:
Maybe we should also mention that the `ApplicationShell` component already includes an instance of this component so pages will automatically get a new title when using the next `app-kit`'s version (we would add here the specific version where this functionality will be available).

Reply:
Thank you, updated here: https://github.com/commercetools/merchant-center-application-kit/pull/2793/commits/06c62b0dc6752af0edc73e85e48faf8a0639d888